clarify access rights of emeritus members (#10)
conda-forge/governance: README.md

  at least follow the same contacting methods stated for Time-out Quorum Rules.
  Any core member can also request to become emeritus if they wish to do so
  (e.g. taking a sabbatical or long vacation). Emeritus core members can still

>   vote and move back to active core anytime, but do not remain "owners" of the
>   conda-forge github organization. Emeritus votes are used to count

Reviewer: I would word this in a service-agnostic way, something like:

```suggestion
  vote and move back to active core anytime, but will not retain privileged access to any conda-forge service. Emeritus votes are used to count
```

This document was written by Anthony Scopatz.

In April 2025, two new clauses were added regarding core and core-emeritus
membership.

> In September 2025, a new clauses was added to clarify access rights of emeritus members.

Reviewer: ```suggestion
In September 2025, a new clause was added to clarify access rights of emeritus members.
```
